src/main/java/com/musala/atmosphere/client/uiutils/CssToXPathConverter.java

```diff
@@ -145,31 +173,34 @@ public static String convertCssToXPath(String cssQuery) throws InvalidCssQueryEx
             if (partOfCssQuery.matches(REGEX_FOR_CSS_QUERY_WITH_CONTAINS)) {
                 attributeNameAndSelectionExpression = partOfCssQuery.split("\\*=");
 
-                attributeName = attributeNameAndSelectionExpression[0];
-                selectionExpression = attributeNameAndSelectionExpression[1];
-
-                if (isAttributeStringOfTheEnumeration(attributeName)) {
+                try {
```

Comment:
You have this logic multiple times. It will be better to move it in a private method.